easybuild/easyconfigs/a/AdapterRemoval/AdapterRemoval-2.3.2-foss-2020b.eb

```
@@ -0,0 +1,39 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'ConfigureMake'
+
+name = 'AdapterRemoval'
+version = '2.3.2'
+
+github_account = 'MikkelSchubert'
+homepage = 'https://github.com/%/(github_account)s/%(namelower)s'
```

It is convenient to use explicit URLs in the homepage, that helps copy pasting from the easyconfig file
```suggestion
homepage = 'https://github.com/MikkelSchubert/adapterremoval'
```